src/Components/Facility/CoverImageEditModal.tsx

```diff
@@ -283,9 +286,19 @@ const CoverImageEditModal = ({
                 <ButtonV2
                   variant="danger"
                   onClick={handleDelete}
-                  disabled={isUploading}
+                  disabled={isDeleting}
                 >
-                  {t("delete")}
+                  {isDeleting ? (
+                    <CareIcon
+                      icon="l-spinner"
+                      className="animate-spin text-lg"
+                    />
+                  ) : (
+                    <CareIcon icon="l-save" className="text-lg" />
+                  )}
+                  <span>
+                    {isDeleting ? `${t("Deleting")}...` : `${t("delete")}`}
```

Comment:
Don't we have something simlar to [disabling cover](https://github.com/pupilfirst/pupilfirst/blob/master/app/frontend/shared/components/DisablingCover.res) that disables the entrie action on save?

Reply:
i just used what we already do for upload we just wanted a re-render to happen here in order to fix the bug should i remove it ?